### PR TITLE
Magento_Bundle: avoid using deprecated escape* methods from AbstractB…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE, Integration Tests, Static Tests

File: app/code/Magento/Bundle/view/adminhtml/templates/catalog/product/edit/tab/attributes/extend.phtml [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 /** @var $block \Magento\Bundle\Block\Adminhtml\Catalog\Product\Edit\Tab\Attributes\Extend */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
+/** @var \Magento\Framework\Escaper $escaper */
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/catalog/product/edit/tab/attributes/extend.phtml]: Could you combine it to a single phpdoc block?

File: app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/bundle.phtml [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 <?php
 /* @var $block \Magento\Bundle\Block\Adminhtml\Catalog\Product\Composite\Fieldset\Bundle */
+/** @var \Magento\Framework\Escaper $escaper */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/bundle.phtml]: Could you combine it to a single phpdoc block?

File: app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/checkbox.phtml [hunk 3]
```diff
@@ -5,14 +5,19 @@
  */
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
-<?php /* @var $block \Magento\Bundle\Block\Adminhtml\Catalog\Product\Composite\Fieldset\Options\Type\Checkbox */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/checkbox.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/checkbox.phtml [hunk 4]
```diff
@@ -5,14 +5,19 @@
...
+/**
+ * @var $block \Magento\Bundle\Block\Adminhtml\Catalog\Product\Composite\Fieldset\Options\Type\Checkbox
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/checkbox.phtml]:
```suggestion
 * @var \Magento\Bundle\Block\Adminhtml\Catalog\Product\Composite\Fieldset\Options\Type\Checkbox $block
```

File: app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/multi.phtml [hunk 5]
```diff
@@ -5,31 +5,36 @@
  */
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
-<?php /* @var $block \Magento\Bundle\Block\Adminhtml\Catalog\Product\Composite\Fieldset\Options\Type\Multi */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/multi.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/multi.phtml [hunk 6]
```diff
@@ -5,31 +5,36 @@
...
+/**
+ * @var $block \Magento\Bundle\Block\Adminhtml\Catalog\Product\Composite\Fieldset\Options\Type\Multi
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/multi.phtml]:
```suggestion
 * @var \Magento\Bundle\Block\Adminhtml\Catalog\Product\Composite\Fieldset\Options\Type\Multi $block
```

File: app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/radio.phtml [hunk 7]
```diff
@@ -5,71 +5,76 @@
  */
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
-<?php /* @var $block \Magento\Bundle\Block\Adminhtml\Catalog\Product\Composite\Fieldset\Options\Type\Radio */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/radio.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/select.phtml [hunk 8]
```diff
@@ -5,35 +5,40 @@
  */
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
-<?php /* @var $block \Magento\Bundle\Block\Adminhtml\Catalog\Product\Composite\Fieldset\Options\Type\Select */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/select.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/Bundle/view/adminhtml/templates/product/edit/bundle.phtml [hunk 9]
```diff
@@ -6,6 +6,7 @@
 
 /** @var $block \Magento\Bundle\Block\Adminhtml\Catalog\Product\Edit\Tab\Bundle */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
+/** @var \Magento\Framework\Escaper $escaper */
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/product/edit/bundle.phtml]: Could you combine it to a single phpdoc block?

File: app/code/Magento/Bundle/view/adminhtml/templates/product/edit/bundle/option.phtml [hunk 10]
```diff
@@ -6,14 +6,15 @@
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 /** @var $block \Magento\Bundle\Block\Adminhtml\Catalog\Product\Edit\Tab\Bundle\Option */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
+/** @var \Magento\Framework\Escaper $escaper */
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/product/edit/bundle/option.phtml]: Could you combine it to a single phpdoc block?

File: app/code/Magento/Bundle/view/adminhtml/templates/product/edit/bundle/option/selection.phtml [hunk 11]
```diff
@@ -6,22 +6,23 @@
 
 /** @var $block \Magento\Bundle\Block\Adminhtml\Catalog\Product\Edit\Tab\Bundle\Option\Selection */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
+/** @var \Magento\Framework\Escaper $escaper */
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/product/edit/bundle/option/selection.phtml]: Could you combine it to a single phpdoc block?

File: app/code/Magento/Bundle/view/adminhtml/templates/sales/creditmemo/create/items/renderer.phtml [hunk 12]
```diff
@@ -35,7 +35,7 @@ $catalogHelper = $block->getData('catalogHelper');
         <?php if ($_prevOptionId != $attributes['option_id']): ?>
         <tr>
             <td class="col-product">
-                <div class="option-label"><?= $block->escapeHtml($attributes['option_label']) ?></div>
+                <div class="option-label"><?= $escaper->escapeHtml($attributes['option_label']) ?></div>
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/sales/creditmemo/create/items/renderer.phtml:38]: It seems like you missed defining escaper in this file

File: app/code/Magento/Bundle/view/adminhtml/templates/sales/creditmemo/view/items/renderer.phtml [hunk 13]
```diff
@@ -35,7 +35,7 @@ $catalogHelper = $block->getData('catalogHelper');
         <?php if ($_prevOptionId != $attributes['option_id']): ?>
         <tr>
             <td class="col-product">
-                <div class="option-label"><?= $block->escapeHtml($attributes['option_label']) ?></div>
+                <div class="option-label"><?= $escaper->escapeHtml($attributes['option_label']) ?></div>
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/sales/creditmemo/view/items/renderer.phtml:37]: It seems like you missed defining escaper in this file

File: app/code/Magento/Bundle/view/adminhtml/templates/sales/invoice/create/items/renderer.phtml [hunk 14]
```diff
@@ -45,7 +45,7 @@ $catalogHelper = $block->getData('catalogHelper');
         <?php if ($_prevOptionId != $attributes['option_id']): ?>
         <tr>
             <td class="col-product">
-                <div class="option-label"><?= $block->escapeHtml($attributes['option_label']) ?></div>
+                <div class="option-label"><?= $escaper->escapeHtml($attributes['option_label']) ?></div>
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/sales/invoice/create/items/renderer.phtml:47]: It seems like you missed defining escaper in this file

File: app/code/Magento/Bundle/view/adminhtml/templates/sales/invoice/view/items/renderer.phtml [hunk 15]
```diff
@@ -35,7 +35,7 @@ $catalogHelper = $block->getData('catalogHelper');
         <?php if ($_prevOptionId != $attributes['option_id']): ?>
         <tr>
             <td class="col-product">
-                <div class="option-label"><?= $block->escapeHtml($attributes['option_label']) ?></div>
+                <div class="option-label"><?= $escaper->escapeHtml($attributes['option_label']) ?></div>
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/sales/invoice/view/items/renderer.phtml:37]: It seems like you missed defining escaper in this file

File: app/code/Magento/Bundle/view/adminhtml/templates/sales/order/view/items/renderer.phtml [hunk 16]
```diff
@@ -35,7 +35,7 @@ $catalogHelper = $block->getData('catalogHelper');
         <?php if ($_prevOptionId != $attributes['option_id']): ?>
         <tr>
             <td class="col-product">
-                <div class="option-label"><?= $block->escapeHtml($attributes['option_label']) ?></div>
+                <div class="option-label"><?= $escaper->escapeHtml($attributes['option_label']) ?></div>
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/sales/order/view/items/renderer.phtml:37]: It seems like you missed defining escaper in this file

File: app/code/Magento/Bundle/view/adminhtml/templates/sales/shipment/create/items/renderer.phtml [hunk 17]
```diff
@@ -32,7 +32,7 @@ $catalogHelper = $block->getData('catalogHelper');
         <?php if ($_prevOptionId != $attributes['option_id']): ?>
         <tr>
             <td class="col-product">
-                <div class="option-label"><?= $block->escapeHtml($attributes['option_label']) ?></div>
+                <div class="option-label"><?= $escaper->escapeHtml($attributes['option_label']) ?></div>
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/sales/shipment/create/items/renderer.phtml:35]: It seems like you missed defining escaper in this file

File: app/code/Magento/Bundle/view/adminhtml/templates/sales/shipment/view/items/renderer.phtml [hunk 18]
```diff
@@ -33,7 +33,7 @@ $catalogHelper = $block->getData('catalogHelper');
         <?php if ($_prevOptionId != $attributes['option_id']): ?>
         <tr>
             <td class="col-product">
-                <div class="option-label"><?= $block->escapeHtml($attributes['option_label']) ?></div>
+                <div class="option-label"><?= $escaper->escapeHtml($attributes['option_label']) ?></div>
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/adminhtml/templates/sales/shipment/view/items/renderer.phtml:38]: It seems like you missed defining escaper in this file

File: app/code/Magento/Bundle/view/base/templates/product/price/tier_prices.phtml [hunk 19]
```diff
@@ -6,14 +6,17 @@
 ?>
 
 <?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/base/templates/product/price/tier_prices.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/backbutton.phtml [hunk 20]
```diff
@@ -3,8 +3,11 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+/**
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/backbutton.phtml:7]:
```suggestion

/**
```

File: app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/customize.phtml [hunk 21]
```diff
@@ -3,6 +3,9 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+/**
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/customize.phtml:7]:
```suggestion

/**
```

File: app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/summary.phtml [hunk 22]
```diff
@@ -3,6 +3,9 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+/**
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/summary.phtml:7]:
```suggestion

/**
```

File: app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle.phtml [hunk 23]
```diff
@@ -4,17 +4,20 @@
  * See COPYING.txt for license details.
  */
 
-/* @var $block \Magento\Bundle\Block\Catalog\Product\View\Type\Bundle */
+/**
+ * @var $block \Magento\Bundle\Block\Catalog\Product\View\Type\Bundle
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle.phtml]:
```suggestion
 * @var \Magento\Bundle\Block\Catalog\Product\View\Type\Bundle $block
```

File: app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/option/checkbox.phtml [hunk 24]
```diff
@@ -3,59 +3,62 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+/**
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/option/checkbox.phtml:7]:
```suggestion

/**
```

File: app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/option/multi.phtml [hunk 25]
```diff
@@ -4,33 +4,38 @@
  * See COPYING.txt for license details.
  */
 ?>
-<?php /* @var $block \Magento\Bundle\Block\Catalog\Product\View\Type\Bundle\Option\Multi */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/option/multi.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/option/multi.phtml [hunk 26]
```diff
@@ -4,33 +4,38 @@
...
+/**
+ * @var $block \Magento\Bundle\Block\Catalog\Product\View\Type\Bundle\Option\Multi
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/option/multi.phtml]:
```suggestion
 * @var \Magento\Bundle\Block\Catalog\Product\View\Type\Bundle\Option\Multi $block
```

File: app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/option/radio.phtml [hunk 27]
```diff
@@ -4,15 +4,20 @@
  * See COPYING.txt for license details.
  */
 ?>
-<?php /* @var $block \Magento\Bundle\Block\Catalog\Product\View\Type\Bundle\Option\Radio */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/option/radio.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/option/radio.phtml [hunk 28]
```diff
@@ -4,15 +4,20 @@
...
+/**
+ * @var $block \Magento\Bundle\Block\Catalog\Product\View\Type\Bundle\Option\Radio
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/option/radio.phtml]:
```suggestion
 * @var \Magento\Bundle\Block\Catalog\Product\View\Type\Bundle\Option\Radio $block
```

File: app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/option/select.phtml [hunk 29]
```diff
@@ -5,42 +5,47 @@
  */
 ?>
 
-<?php /* @var $block \Magento\Bundle\Block\Catalog\Product\View\Type\Bundle\Option\Select */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/option/select.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/option/select.phtml [hunk 30]
```diff
@@ -5,42 +5,47 @@
...
+/**
+ * @var $block \Magento\Bundle\Block\Catalog\Product\View\Type\Bundle\Option\Select
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/option/select.phtml]:
```suggestion
 * @var \Magento\Bundle\Block\Catalog\Product\View\Type\Bundle\Option\Select $block
```

File: app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/options.phtml [hunk 31]
```diff
@@ -5,7 +5,12 @@
  */
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
-<?php /** @var $block Magento\Bundle\Block\Catalog\Product\View\Type\Bundle */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/options.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/options.phtml [hunk 32]
```diff
@@ -5,7 +5,12 @@
...
+/**
+ * @var $block Magento\Bundle\Block\Catalog\Product\View\Type\Bundle
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/options.phtml]:
```suggestion
 * @var Magento\Bundle\Block\Catalog\Product\View\Type\Bundle $block
```

File: app/code/Magento/Bundle/view/frontend/templates/email/order/items/creditmemo/default.phtml [hunk 33]
```diff
@@ -5,7 +5,12 @@
  */
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
-<?php /** @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/email/order/items/creditmemo/default.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/Bundle/view/frontend/templates/email/order/items/creditmemo/default.phtml [hunk 34]
```diff
@@ -5,7 +5,12 @@
...
+/**
+ * @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/email/order/items/creditmemo/default.phtml]:
```suggestion
 * @var \Magento\Bundle\Block\Sales\Order\Items\Renderer $block
```

File: app/code/Magento/Bundle/view/frontend/templates/email/order/items/invoice/default.phtml [hunk 35]
```diff
@@ -5,7 +5,12 @@
  */
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
-<?php /** @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/email/order/items/invoice/default.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/Bundle/view/frontend/templates/email/order/items/invoice/default.phtml [hunk 36]
```diff
@@ -5,7 +5,12 @@
...
+/**
+ * @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/email/order/items/invoice/default.phtml]:
```suggestion
 * @var \Magento\Bundle\Block\Sales\Order\Items\Renderer $block
```

File: app/code/Magento/Bundle/view/frontend/templates/email/order/items/order/default.phtml [hunk 37]
```diff
@@ -5,7 +5,12 @@
  */
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
-<?php /** @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/email/order/items/order/default.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/Bundle/view/frontend/templates/email/order/items/order/default.phtml [hunk 38]
```diff
@@ -5,7 +5,12 @@
...
+/**
+ * @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/email/order/items/order/default.phtml]:
```suggestion
 * @var \Magento\Bundle\Block\Sales\Order\Items\Renderer $block
```

File: app/code/Magento/Bundle/view/frontend/templates/email/order/items/shipment/default.phtml [hunk 39]
```diff
@@ -5,7 +5,12 @@
  */
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
-<?php /** @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/email/order/items/shipment/default.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/Bundle/view/frontend/templates/email/order/items/shipment/default.phtml [hunk 40]
```diff
@@ -5,7 +5,12 @@
...
+/**
+ * @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/email/order/items/shipment/default.phtml]:
```suggestion
 * @var \Magento\Bundle\Block\Sales\Order\Items\Renderer $block
```

File: app/code/Magento/Bundle/view/frontend/templates/sales/order/creditmemo/items/renderer.phtml [hunk 41]
```diff
@@ -5,7 +5,12 @@
  */
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
-<?php /** @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/sales/order/creditmemo/items/renderer.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/Bundle/view/frontend/templates/sales/order/creditmemo/items/renderer.phtml [hunk 42]
```diff
@@ -5,7 +5,12 @@
...
+/**
+ * @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/sales/order/creditmemo/items/renderer.phtml]:
```suggestion
 * @var \Magento\Bundle\Block\Sales\Order\Items\Renderer $block
```

File: app/code/Magento/Bundle/view/frontend/templates/sales/order/invoice/items/renderer.phtml [hunk 43]
```diff
@@ -26,42 +26,42 @@
         <?php if ($_prevOptionId != $attributes['option_id']) : ?>
             <tr class="options-label">
                 <td class="col label" colspan="5">
-                    <div class="option label"><?= $block->escapeHtml($attributes['option_label']) ?></div>
+                    <div class="option label"><?= $escaper->escapeHtml($attributes['option_label']) ?></div>
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/sales/order/invoice/items/renderer.phtml:34]: It seems like you missed defining escaper in this file

File: app/code/Magento/Bundle/view/frontend/templates/sales/order/items/renderer.phtml [hunk 44]
```diff
@@ -4,7 +4,10 @@
  * See COPYING.txt for license details.
  */
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
-/** @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer */
+/**
+ * @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/sales/order/items/renderer.phtml]:
```suggestion
 * @var \Magento\Bundle\Block\Sales\Order\Items\Renderer $block
```

File: app/code/Magento/Bundle/view/frontend/templates/sales/order/shipment/items/renderer.phtml [hunk 45]
```diff
@@ -5,7 +5,12 @@
  */
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
-<?php /** @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/sales/order/shipment/items/renderer.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/Bundle/view/frontend/templates/sales/order/shipment/items/renderer.phtml [hunk 46]
```diff
@@ -5,7 +5,12 @@
...
+/**
+ * @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/sales/order/shipment/items/renderer.phtml]:
```suggestion
 * @var \Magento\Bundle\Block\Sales\Order\Items\Renderer $block
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Bundle/view/frontend/templates/sales/order/shipment/items/renderer.phtml (hunk 45 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Bundle/view/frontend/templates/sales/order/shipment/items/renderer.phtml]: fixed

**Comment by sergeynezbritskiy** [comment]
Hello @ihor-sviziev I've fixed the issues you mentioned above, inluding templates, not affected by introducing $escaper in templates, please review

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Database Compare

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8794](https://jira.corp.magento.com/browse/ENGCOM-8794) has been created to process this Pull Request

**Pull request closed without merging**